flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/TableScanTest.scala

```diff
@@ -76,6 +76,27 @@ class TableScanTest extends TableTestBase {
     util.verifyPlan("SELECT * FROM t1")
```

Comment:
Could you add plan tests for projection pushdown and metadata pushdown?

Reply:
Very good point! I even found a bug.